src/plugins/console/public/plugin.ts

```diff
@@ -55,14 +54,20 @@ export class ConsoleUIPlugin implements Plugin<any, any> {
         defaultMessage: 'Console',
       }),
       enableRouting: false,
-      async mount({ core: { docLinks } }, { element }) {
+      mount: async ({ core: { docLinks, i18n: i18nDep } }, { element }) => {
         const { boot } = await import('./application');
+        const [{ injectedMetadata }] = await getStartServices();
+        const elasticsearchUrl = injectedMetadata.getInjectedVar(
+          'elasticsearchUrl',
+          'http://localhost:9200'
+        ) as string;
```

Comment:
Guess I'm a bit surprised we expose this to the client. Is this necessary? Seems like a potential low-severity security issue? If it can be removed, can be addressed separately from this PR.

Reply:
Yeah, it is a bit awkward :c. The reason it exists is for the copy as cURL functionality in console. So that we can provide a better experience for going from Console to the terminal.

I agree that this qualifies as low-severity security concern. Do you think it merits being removed for that reason? To be clear, my understanding of the security issue is that we are leaking DNS entry + port to client side and can't assume we are in a secured environment (like a VPC)?